src/components/Editor/Runners/PythonRunner/PyodideRunner/PyodideRunner.jsx

```diff
@@ -106,25 +114,20 @@ const PyodideRunner = (props) => {
     }
   }, [codeRunTriggered]);
 
-  useEffect(() => {
-    if (codeRunTriggered) {
-      setShowRunner(active);
-    }
-  }, [codeRunTriggered]);
-
   useEffect(() => {
     if (codeRunStopped && active) {
       handleStop();
     }
   }, [codeRunStopped]);
 
   const handleLoading = () => {
-    dispatch(loadingRunner());
+    return;
```

Comment:
I don't know the reason behind this change, so just checking we want this here?

Reply:
ah! So I've hijacked  the`loadingRunner` action a level higher in the `PythonRunner` to set the "requested" runner and then `setLoadedRunner` is used to set the `loadedRunner`. So currently there's no use for a loading method in the pyodide runner itself but I figured its useful to know that that method gets posted back from the worker 😅